### PR TITLE
Update new release info (EXPOSUREAPP-4731) 

### PR DESCRIPTION
**Pull request by chiljamgossow**
update layout
use recycler view
update content
dark mode illustration
unit test
device-2021-01-29-103501

### CODE REVIEW

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @Test
+    fun testOnNextButtonClick() {
+        viewModel.onNextButtonClick()
+        viewModel.routeToScreen.value shouldBe NewReleaseInfoNavigationEvents.CloseScreen
+    }
+
+    @Test
+    fun testCountryList() {
```
Comment (d4rken) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt:39]: Why is this called `countryList`?

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import io.mockk.impl.annotations.MockK
+import io.mockk.just
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import testhelpers.TestDispatcherProvider
+import testhelpers.extensions.InstantExecutorExtension
+
+@ExtendWith(InstantExecutorExtension::class)
+class NewReleaseInfoViewModelTest {
```
Comment (d4rken) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt:17]: Could you add a test for what happens if there is a missmatch between titles and bodies?

**Review by d4rken — COMMENTED** [review]
Looks good, could I have an extra test?

File: Corona-Warn-App/src/main/res/layout/new_release_info_item.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <ImageView
+            android:id="@+id/bullet_point"
+            android:layout_width="@dimen/bullet_point_size"
+            android:layout_height="@dimen/bullet_point_size"
+            android:layout_marginStart="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_tiny"
+            android:importantForAccessibility="no"
+            android:src="@drawable/bullet_point"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@id/title" />
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/layout/new_release_info_item.xml:30]: Could it be that the previous layout had more bottom padding between the graphic and the title? Can you compare with mock ups?

Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/layout/new_release_info_item.xml:30]: I'm not sure if I understand what you mean. In the previous version the bullet point was centered, but that does not work with two-line headings.

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt (hunk 1 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt]: Copy-paste error. Renamed.

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt (hunk 2 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt:17]: 2 more tests added

File: Corona-Warn-App/src/main/res/layout/new_release_info_item.xml (hunk 3 above)
Comment (d4rken) [reply Corona-Warn-App/src/main/res/layout/new_release_info_item.xml:30]: I don't think either looks bad, but I'm not sure if the change was intentional.

Screenshot from 2021-01-29 15-17-41
Screenshot from 2021-01-29 15-18-04

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 4]
```diff
@@ -5,20 +5,34 @@
...
+        <item>Anpassungen der Risiko-Karten</item>
+        <item>Fortführung der Risiko-Ermittlung nach Teilung der Zufalls-IDs</item>
+        <item>Einführung von Registerkarten</item>
+        <item>Mehr Informationen zum Testablauf</item>
+    </string-array>
+
+    <!-- XTXT: Text bodies for the release info screen bullet points -->
+    <string-array name="new_release_body">
+        <item>Wenn Sie ein erhöhtes Risiko haben, können Sie aus der App heraus eine Umfrage des Robert-Koch-Instituts aufrufen und daran teilnehmen.</item>
+        <item>Sie bekommen einen Überblick mit aktuellen Kennzahlen zum Infektionsgeschehen in Deutschland.</item>
```
Comment (fynngodau) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:31]: That text doesn't quite fit the heading "Einwilligung zur Datenspende", does it? It sounds more like it is intended to be used with the heading "Statistiken", and it matches the removed string `release_info_1_12_statistics_body`.

Comment (SabineLoss) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:31]: Yes. This item needs to be removed.

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 5]
```diff
@@ -5,20 +5,34 @@
...
+        <item>Die Texte auf den Risiko-Karten wurden leicht angepasst.</item>
+        <item>Nachdem Sie Ihre verschlüsselten Zufalls-IDs geteilt haben können Sie entscheiden, ob und wann Sie die Risiko-Ermittlung wieder einschalten möchten.</item>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:33]:
```suggestion
        <item>Nachdem Sie Ihre verschlüsselten Zufalls-IDs geteilt haben, können Sie entscheiden, ob und wann Sie die Risiko-Ermittlung wieder einschalten.</item>
```

File: Corona-Warn-App/src/main/res/values/release_info_strings.xml [hunk 6]
```diff
@@ -5,20 +5,34 @@
...
+        <item>Einführung von Registerkarten</item>
+        <item>Mehr Informationen zum Testablauf</item>
+    </string-array>
+
+    <!-- XTXT: Text bodies for the release info screen bullet points -->
+    <string-array name="new_release_body">
+        <item>Wenn Sie ein erhöhtes Risiko haben, können Sie aus der App heraus eine Umfrage des Robert-Koch-Instituts aufrufen und daran teilnehmen.</item>
+        <item>Sie bekommen einen Überblick mit aktuellen Kennzahlen zum Infektionsgeschehen in Deutschland.</item>
+        <item>Die Texte auf den Risiko-Karten wurden leicht angepasst.</item>
+        <item>Nachdem Sie Ihre verschlüsselten Zufalls-IDs geteilt haben können Sie entscheiden, ob und wann Sie die Risiko-Ermittlung wieder einschalten möchten.</item>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values/release_info_strings.xml:33]:
```suggestion
        <item>Nachdem Sie Ihre verschlüsselten Zufalls-IDs geteilt haben, können Sie entscheiden, ob und wann Sie die Risiko-Ermittlung wieder einschalten.</item>
```

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml (hunk 4 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:31]: Exchanged texts

File: Corona-Warn-App/src/main/res/layout/new_release_info_item.xml (hunk 3 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/layout/new_release_info_item.xml:30]: yes, I updated the illustration, so both have the same dimensions.

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml (hunk 5 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml]: adjusted the texts

File: Corona-Warn-App/src/main/res/values/release_info_strings.xml (hunk 6 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/values/release_info_strings.xml]: adjusted text

**Review by d4rken — APPROVED** [review]
Fantastic :ok_hand:

**Review by SamuraiKek — APPROVED** [review]
LGTM. Tested on Huawei Mate 8 (Android 6) and Nokia 3 (Android 9)

**Review by SabineLoss — APPROVED** [review]
UA reviewed

**Pull request merged**